forc-util/src/lib.rs

```diff
@@ -407,6 +409,25 @@ fn construct_window<'a>(
     &input[calculated_start_ix..calculated_end_ix]
 }
 
+const LOG_FILTER: &str = "RUST_LOG";
+
+pub fn init_tracing_subscriber() {
```

Comment:
Would you mind adding a small doc comment for future readers briefly describing how this subscriber differs from the default and why we do this?